lib/components/viewers/nearby/nearby-view.tsx

```diff
@@ -326,6 +351,20 @@ const mapStateToProps = (state: AppReduxState) => {
       ? getCurrentServiceWeek()
       : undefined
 
+  // TODO: Refine so we don't have this same thing in stops.tsx
+  // Default sort: departure time
+  let routeSortComparator = (a: PatternStopTime, b: PatternStopTime) =>
+    fullTimestamp(a.stoptimes?.[0]) - fullTimestamp(b.stoptimes?.[0])
+
+  if (nearbyViewConfig?.useRouteViewSort) {
+    routeSortComparator = (a: PatternStopTime, b: PatternStopTime) =>
+      coreUtils.route.makeRouteComparator(transitOperators)(
+        // @ts-expect-error core-utils types are wrong!
```

Comment:
are these types important?

Reply:
I admit I copied this verbatim from `stops.tsx` but it looks like if we need to update this it would be in OTP-UI